sdks/java/core/src/test/java/org/apache/beam/sdk/util/GcsUtilTest.java

```diff
@@ -321,28 +321,72 @@ public void testAccessDeniedObjectThrowsIOException() throws IOException {
 
   @Test
   public void testGetSizeBytes() throws Exception {
-    GcsOptions pipelineOptions = gcsOptionsWithTestCredential();
-    GcsUtil gcsUtil = pipelineOptions.getGcsUtil();
-
-    Storage mockStorage = Mockito.mock(Storage.class);
-    gcsUtil.setStorageClient(mockStorage);
-
-    Storage.Objects mockStorageObjects = Mockito.mock(Storage.Objects.class);
-    Storage.Objects.Get mockStorageGet = Mockito.mock(Storage.Objects.Get.class);
-
-    when(mockStorage.objects()).thenReturn(mockStorageObjects);
-    when(mockStorageObjects.get("testbucket", "testobject")).thenReturn(mockStorageGet);
-    when(mockStorageGet.execute()).thenReturn(
-        new StorageObject().setSize(BigInteger.valueOf(1000)));
-
-    assertEquals(1000, gcsUtil.fileSize(GcsPath.fromComponents("testbucket", "testobject")));
+    String singleFile = "--batch_foobarbaz\n"
+        + "Content-Type: application/http\n"
+        + "\n"
+        + "HTTP/1.1 200 OK\n"
+        + "Content-Length: 708\n"
+        + "\n"
+        + "{\n"
+        + " \"kind\": \"storage#object\",\n"
+        + " \"id\": \"apache-beam-samples/apache/LICENSE/1470763581808000\",\n"
+        + " \"selfLink\": \"https://www.googleapis.com/storage/v1/b/apache-beam-samples/o/apache%2FLICENSE\",\n"
+        + " \"name\": \"apache/LICENSE\",\n"
+        + " \"bucket\": \"apache-beam-samples\",\n"
+        + " \"generation\": \"1470763581808000\",\n"
+        + " \"metageneration\": \"2\",\n"
+        + " \"contentType\": \"application/txt\",\n"
+        + " \"timeCreated\": \"2016-08-09T17:26:21.801Z\",\n"
```

Comment:
This is very overspecified. Anything that could change between requests, such as creation time, etc, should not be included.